data-prepper-plugins/opensearch/README.md

```diff
@@ -74,6 +74,10 @@ Default is null.
 
 - `insecure`: A boolean flag to turn off SSL certificate verification. If set to true, CA certificate verification will be turned off and insecure HTTP requests will be sent. Default to `false`.
 
+- `socket_timeout`(optional): An integer value indicates the timeout in milliseconds for waiting for data (or, put differently, a maximum period inactivity between two consecutive data packets). A timeout value of zero is interpreted as an infinite timeout. A negative value is interpreted as undefined (system default if applicable). Default: -1.
```

Comment:
> A negative value is interpreted as undefined (system default if applicable). Default: -1.

This says that a negative value is undefined and then says the default is negative. 

I know this comes from the Apache documentation. But, this is confusing in the context of the Data Prepper documentation. 

There might be a better way to state this. Maybe even saying that this value is part of the underlying HTTP framework would help.

And what is the system in system default? Data Prepper, the OS? Once I read the Apache HTTP documentation it was clear, because I knew the context better. But here it is a little confusing since the system could be Data Prepper.

Reply:
Good point! 

I re-phrased the sentence to something like "If this timeout value is either negative or not set, the underlying Apache HttpClient would rely on operating system settings for managing connection timeouts",  hoping it's clearer now. Let me know if you have further suggestions. Thanks.